README.md

```diff
@@ -17,38 +17,37 @@ Note all endpoints are ported. If you wish to port any missing endpoints, create
 
 Include the fetch wrapper into your project
 
-```
-<script src="https://cdn.jsdelivr.net/gh/zesty-io/fetch-wrapper@latest/dist/index.min.js" />
+```jsx
+<script src="https://cdn.jsdelivr.net/gh/zesty-io/fetch-wrapper@latest/dist/index.js" />
```

Comment:
@darwin808 apps like nextjs-website, live-editor, and content-composer use this, please let @glespinosa know to update this